src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -28,40 +35,47 @@ public String greeting(@RequestParam(name="name", required=false, defaultValue="
 	@Autowired
 	private BookRepository bookRepository;
 
-	@GetMapping
+	@GetMapping("/")
+	@ApiOperation(value = "Index page")
 	public String index() {
-		return "index";
+			return "index";
 	}
 
 	@GetMapping("/list")
```

Comment:
Sorry I didn't notice this in previous PRs 😓the endpoint should be simply "/" :) 
 Check out the comment about RESTful APIs naming conventions that I wrote in the UserController PR  😄